pkgs/development/libraries/gdal/default.nix

```diff
@@ -20,14 +20,13 @@ composableDerivation.composableDerivation {} (fixed: rec {
 
   hardeningDisable = [ "format" ];
 
-  # Don't use optimization for gcc >= 4.3. That's said to be causing segfaults.
   # Unset CC and CXX as they confuse libtool.
-  preConfigure = "export CFLAGS=-O0 CXXFLAGS=-O0; unset CC CXX";
```

Comment:
@LnL7: The [conda-forge/gdal-feedstock](https://github.com/conda-forge/gdal-feedstock), also doesn't disable any optimizations in the build. I can't really find any reference for the harmfulness of the optimizations. I don't think disabling optimizations is a good option for gdal. If there are segfaults when enabling optimizations this must be due to some parts of the code relying on undefined behaviour, i.e. a bug in gdal. The only thing I found is this: https://trac.osgeo.org/gdal/wiki/SupportedCompilers but it never states anything about optimization levels.